addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -27,6 +27,12 @@
                   class="badge bg-success">
                     <i class="fa fa-check me-1"/>Solved
             </span>
+            <span t-if="question.state=='pending'" 
```

Comment:
If posts cannot be solved and pending, we could use `t-elif`.
Also, please mind the python styling guidelines even in `t-` instructions.